### ISSUE TITLE
Korrigierte Zahlen einbinden

### ISSUE THREAD

**Issue by errotu** [eknoes/covidbot#221]
Per E-Mail kam die Anfrage, ob wir die korrigierten Zahlen mit Nachmeldungen nicht doch (zusätzlich) in den Grafiken darstellen wollen, um ein realistischeres Bild der aktuellen Lage zu zeichnen:

> wäre es da nicht ein guter, brauchbarer Kompromiss eine 2. Farbe mit den Nachmeldungen ein zu bauen ?
Ich finde so wird den Städten, durch die falsche Wahrnehmung, wieder mal "in die Hände gespielt".
Die berufen sich ja jetzt mit ihren Maßnamen auf die "falschen" Daten. Wenn man das aber visualisieren könnte und ihnen vor die Nase hält (z.B. via Twitter). Könnte man sie ein bischen "zu Ordnung rufen". Denn gerade bei Köln fällt das extrem auf. Ich weis natürlich nicht, wie das bei den Städten der Fall ist, wo die Inzedenzen noch deutlich höher sind.

**Comment by eknoes** [comment]
Für die Neuinfektionen, oder bezieht sich das auf die Hospitalisierung?

**Comment by errotu** [comment]
Bezieht sich auf die Neuinfektionen.

**Comment by errotu** [comment]
Wir wurden nun per E-Mail nochmal angefragt, ob wir aufgrund der aktuell schlechten Datenlage nicht doch die Nachmeldungen berücksichtigen möchten. Zur Illustration wurde diese beiden Screenshots für Köln angehängt.

Ich sehe den Punkt, dass die aktuellen Zahlen bei weitem nicht so sehr die Realität abbilden, wie es sein sollte. Aber wenn wir nachgemeldete und erstgemeldete Zahlen in die gleiche Grafik packen, vergleichen wir eben auch Äpfel mit Birnen. Vielleicht wäre ein spezieller Befehl für Nachmeldungen denkbar? Oder doch die oben schon skizzierte zweite Farbe (ggf. über eine Option an- und abwählbar). Was denkst du, @eknoes?

**Comment by eknoes** [comment]
Oder wir überschreiben doch die alten Daten mit den nachgemeldeten, ab bspw. einer Woche, und stellen dann immer die letzte Woche in einer anderen Farbe da? Ich glaube sonst würde ich eher einen neuen Befehl einbauen.

**Comment by errotu** [comment]
Ich fände es nicht so gut, wenn die ursprünglich gemeldeten Zahlen komplett wegfallen. Gehe davon aus, dass das (wieder zu Verwirrung führt). Wäre es nicht vielleicht möglich, dass die Säule der ursprünglich gemeldeten Zahlen einen farblichen "Aufsatz" bekommt, der die darüber hinaus gemeldeten nachgemeldeten Infektionen anzeigt? Ähnlich wie wir es bei den Impfungen machen, nur als Säulendiagramm.

**Comment by eknoes** [comment]
Ja, das ist eine gute Idee. Ich weiß nur nicht, wann ich das schaffe. Es gibt ein Matplotlib-Beispiel für Stacked Bar Charts (https://matplotlib.org/stable/gallery/lines_bars_and_markers/bar_stacked.html#sphx-glr-gallery-lines-bars-and-markers-bar-stacked-py), das heißt das wäre auch nicht so schwierig umzusetzen. Dann brauchen wir nur noch die Daten, die die Nachmeldungen enthalten.
